WalletWasabi/Blockchain/Transactions/TransactionFactory.cs

```diff
@@ -296,5 +300,44 @@ public TransactionFactory(Network network, KeyManager keyManager, ICoinsView coi
 			var spendsUnconfirmed = spentCoins.Any(c => !c.Confirmed);
 			return new BuildTransactionResult(new SmartTransaction(tx, Height.Unknown), psbt, spendsUnconfirmed, sign, fee, feePc, outerWalletOutputs, innerWalletOutputs, spentCoins);
 		}
+
+		private PSBT TryNegotiatePayjoin(IPayjoinClient payjoinClient, TransactionBuilder builder, PSBT psbt)
+		{
+			try
+			{
+				psbt = payjoinClient.RequestPayjoin(psbt,
+					KeyManager.ExtPubKey,
+					new RootedKeyPath(KeyManager.MasterFingerprint.Value, KeyManager.DefaultAccountKeyPath),
+					CancellationToken.None).GetAwaiter().GetResult();
+				builder.SignPSBT(psbt);
+			}
+			catch (HttpRequestException e)
+			{
+				Logger.LogWarning($"Payjoin server responded with {e.ToTypeMessageString()}. Ignoring...");
+			}
+			catch (PayjoinException e)
+			{
+				Logger.LogWarning($"Payjoin server responded with {e.Message}. Ignoring...");
```

Comment:
Why are we shallowing the exception here? If the PayJoin cannot be done, shouldn't we abort the whole tx?

Reply:
No, the show must go on anyway. In the Payjoin connot be done we must send the normal transaction anyway. 
Something to keep in mind is that we have sent a signed tx to the server so, we have already pay them and they can broadcast the transaction anytime so, it doesn't matter if they response with a 500, 503, 404, 400 or server unreachable, we pay anyway and broadcast the transaction with no exception.